examples/google-vertex-system-instruction/promptfooconfig.yaml

```
@@ -0,0 +1,34 @@
+        responseMimeType: 'application/json'
+      systemInstruction: file://system-instruction.txt
+
+prompts:
+  - "What is the capital of France?"
+  - "Tell me a joke"
+
+tests:
+  - vars:
+      topic: "artificial intelligence"
+  - vars:
+      topic: "quantum computing"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)